TimeTrackerApp/TimeTrackerApp/ViewController/Tabbar/CustomTabbarViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CustomTabbarViewController.swift
+//  TimeTrackerApp
+//
+//  Created by Sevda Gul Baran on 14.09.2022.
+//
+
+import UIKit
+import FloatingTabBarController
+class CustomTabbarViewController: FloatingTabBarController {
```

Comment:
importtan sonra new line. Eğer inherit edilmesini istemiyorsak başına final yazalım. Performans açısından da etkili

```suggestion
import FloatingTabBarController

final class CustomTabbarViewController: FloatingTabBarController {
```